terminus-ui/chip/src/chip-collection.component.spec.ts

```diff
@@ -67,8 +66,8 @@ describe(`TsChipCollection`, function() {
   let BACKSPACE_EVENT: KeyboardEvent;
   let BACKSPACE_EVENT_CHIP: KeyboardEvent;
 
-  function setupStandardCollection(exampleComponent) {
-    fixture = createComponent(testComponents[exampleComponent]);
+  function setupStandardCollection(component) {
```

Comment:
I thought about an enum here, but realized it wouldn't really get us anything vs what we're doing already.. basically same amount of typing. And using the string hampered the IDE ability to refactor or jump to the definition 😞